src/rpc/rawtransaction.cpp

```diff
@@ -807,14 +807,13 @@ static UniValue sendrawtransaction(const JSONRPCRequest& request)
         max_raw_tx_fee = fr.GetFee((weight+3)/4);
     }
 
-    uint256 txid;
     std::string err_string;
-    const TransactionError err = BroadcastTransaction(tx, txid, err_string, max_raw_tx_fee);
+    const TransactionError err = BroadcastTransaction(tx, err_string, max_raw_tx_fee, true, true);
```

Comment:
87cca293d5b127e43e35d866167893454658eacd

nit, could add a comment on the last 2 args.